examples/cypress/cypress.config.js

```diff
@@ -0,0 +1,16 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  reporter: "../../cypress/reporter",
+  reporterOptions: {
```

Comment:
Maybe this options can be removed? The specs expect that the token can be set via `reporterOptions` and env variable. If we set the token here, it will defeat the purpose of those specs.

Reply:
`reporterOptions` isn't setting the token's value, it's specifying the token's name. So I think this is fine: it's the same thing that the mocha reporter does.